@xen-orchestra/lite/src/components/infra/InfraHostItem.vue

```diff
@@ -18,6 +18,9 @@
           @click="toggle()"
         />
       </template>
+      <li class="vm-number" v-if="!isExpanded && faAngleUp">
```

Comment:
Why did you use an `li` element?
it's supposed to be used to list items. See [MDN](https://developer.mozilla.org/fr/docs/Web/HTML/Element/li)

Reply:
At first I used a span and don't know why i changed it last minute, I'll also add a br tag so it appears under the host and not on the side, is it fine?

Reply:
Don't use `<br />`, use CSS styles instead. (eg: `display: block;`)